ts/src/base/Exchange.ts

```diff
@@ -530,6 +530,7 @@ export default class Exchange {
                 'fetchClosedOrder': undefined,
                 'fetchClosedOrders': undefined,
                 'fetchClosedOrdersWs': undefined,
+                'fetchConvertCurrencies': undefined,
```

Comment:
@Dan-krm two things,
First, can we add a proper return type?
Second, we need to add the stub implementation in the base so the c# recognizes it as an override

Reply:
I'll add the stub and I think we can actually just use the Currencies return type

Reply:
@Dan-krm yeah yeah we can use currencies, just saying that we should not add new untyped methods

Reply:
I'll keep that in mind to make sure the new methods have a type